SPECS-EXTENDED/libgta/libgta.spec

```diff
@@ -0,0 +1,177 @@
+Summary:        Library that implements the Generic Tagged Arrays file format
+Name:           libgta
+Version:        1.2.1
+Release:        9%{?dist}
+License:        LGPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://marlam.de/gta/
+Source0:        https://marlam.de/gta/releases/%{name}-%{version}.tar.xz
+%global so_version 1
```

Comment:
Although the linter does not like it, we generally place all macros at the top.
Ignore the linter if it yells for this.